controlplane/kubeadm/internal/failure_domain.go

```diff
@@ -19,9 +19,13 @@ package internal
 import (
 	"sort"
 
+	"k8s.io/klog/klogr"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 )
 
+// Log is the global logger for the internal package.
+var Log = klogr.New()
```

Comment:
As we move forward we should figure out a better place for this. I think it's too early to bother figuring out right now, but I'm not sure if we want a logger per package or a logger per module.... 🤔 Will be paying attention as this pattern develops.

Reply:
Should this be exported?

Reply:
yeah, this should be exported, that way the other packages can control the logging for this library